plugins/available/vagrant-as-root.plugin.bash

```diff
@@ -0,0 +1,4 @@
+# Simple function for vagrant ssh that immediately switches to root.
```

Comment:
Please take a look at some of the other plugins, e.g. here: https://github.com/Bash-it/bash-it/blob/master/plugins/available/docker-compose.plugin.bash#L1

Multiple comments:
* The name of the plugin should be `vagrant.plugin.bash` - please get rid of the `-as-root` part. Once this file is there, we can use it to add more helper functions for vagrant.
* Please add the file header with the meta tags at the top. This will allow the `bash-it help` function to show what this file is about.